src/main/kotlin/org/openredstone/chad/commands/Command.kt

```diff
@@ -328,7 +328,7 @@ fun deleteCommand(authorizedRoles: List<String>, discordApi: DiscordApi, chadCon
             )
             setColor(Color.RED)
             setFooter("FootORE")
-            setThumbnail("https://cdn.discordapp.com/emojis/892499052942463027.webp")
+            // setThumbnail("https://cdn.discordapp.com/emojis/892499052942463027.webp") // no
```

Comment:
Consider: Yes.

Reply:
fine but accept the other thinggggggggggg pls plslpsplsplplspl